### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by IsraaSulaiman**
#219 

- Fix bugs in queries:
  - InsertTask (assgined_to is the user's full_name, not id)
  - selectProjectDetails: I made inner join with users table to select full_name. @aminalakhsham, check my comments below.
- Fix bugs in tests

### CODE REVIEW

File: server/controllers/projects/getProject.js [hunk 1]
```diff
@@ -5,8 +5,19 @@ exports.getProject = (req, res, next) => {
...
+        description,
+        userNames,
+      } = refactorData(result.rows);
       const projectsData = {
-        data: refactorData(result.rows),
+        data: {
+          id,
+          name,
+          description,
+          userNames,
```
Comment (IsraaSulaiman) [inline server/controllers/projects/getProject.js]: @aminalakhsham, the data structure changed. Sorry, but we had to change ```user_id``` to ```userNames``` which is an array of objects: ```[{user_id: 1, full_name: 'Amin al Akhsham'}, {user_id: 2 , full_name: 'Ahmed Shatat'} ]```

File: server/database/queries/selectProjectDetails.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const connect = require('./../config/connection');
 
 const selectProjectDetails = projectID => connect.query(
-  'SELECT projects.*, user_id, full_name FROM  users_projects INNER JOIN projects ON projects.id = users_projects.project_id INNER JOIN users on users.id = user_id WHERE project_id = $1;',
+  'SELECT projects.*, users.id as user_id, users.full_name FROM users_projects INNER JOIN projects ON projects.id = users_projects.project_id INNER JOIN users on users.id = user_id WHERE project_id = $1;',
```
Comment (IsraaSulaiman) [inline server/database/queries/selectProjectDetails.js]: this is the change we made to the ```selectProjectDetails``` query @aminalakhsham

File: test/server/routes/getProjectRoute.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ connection
             );
             t.deepEqual(
               Object.keys(res.body.data),
-              ['id', 'name', 'description', 'created_at', 'user_id'],
+              ['id', 'name', 'description', 'userNames'],
```
Comment (IsraaSulaiman) [inline test/server/routes/getProjectRoute.js]: I edited the test accordingly @aminalakhsham

File: test/server/routes/insertTaskRoute.js [hunk 4]
```diff
@@ -4,7 +4,7 @@ const route = require('../../../server/app');
 const connect = require('../../../server/database/config/connection');
 
 const selectUserScrumProj = () => {
-  const getUser = 'SELECT projects.*, scrums.project_id as scrum_id, users_projects.user_id as user_id FROM users_projects join projects ON projects.id = users_projects.project_id inner join scrums on scrums.project_id = users_projects.project_id';
```
Comment (IsraaSulaiman) [inline test/server/routes/insertTaskRoute.js]: assigned_to is not the user_id so I changed this query to make the test pass @Angham116

File: server/controllers/projects/getProject.js (hunk 1 above)
Comment (AymanAlqoqa) [reply server/controllers/projects/getProject.js]: this is the main problem with edit project page>>>
I make PR to overcome this

File: test/server/routes/insertTaskRoute.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ const route = require('../../../server/app');
...
+  const getUser = 'select scrums.project_id, scrums.id as scrum_id, users.full_name, users.id as user_id from scrums inner join users_projects on scrums.project_id = users_projects.project_id inner join users on users.id = users_projects.user_id limit 1';
```
Comment (ashatat) [inline test/server/routes/insertTaskRoute.js]: >For readability, avoid lines longer than 80 characters.

[JS_conventions](https://www.w3schools.com/js/js_conventions.asp)

File: server/database/queries/selectProjectDetails.js (hunk 2 above)
Comment (ashatat) [reply server/database/queries/selectProjectDetails.js]: So long line :man_facepalming:

File: test/server/routes/insertTaskRoute.js [hunk 6]
```diff
@@ -46,7 +46,7 @@ test('Testing for api/v1/tasks/new route', (t) => {
   selectUserScrumProj()
     .then((result) => {
       newTask.scrum_id = result.rows[0].scrum_id;
-      newTask.assigned_to = result.rows[0].user_id;
+      newTask.assigned_to = result.rows[0].full_name;
```
Comment (ashatat) [inline test/server/routes/insertTaskRoute.js]: shouldn't this be user_name?

Comment (IsraaSulaiman) [reply test/server/routes/insertTaskRoute.js]: Why user_name? I think full_name is more appropriate in this case. User_names are only for login. What do you think?

File: server/controllers/projects/getProject.js (hunk 1 above)
Comment (IsraaSulaiman) [reply server/controllers/projects/getProject.js]: Good, thanks Ayman

**Pull request merged**